models/Auth.php

```diff
@@ -34,93 +36,102 @@ public function hydrate($data){
 	}
 
 	/**
-	 * @return mixed
+	 * @return int
 	 */
-	public function getId() {
+	public function getId(): int {
 		return $this->_id;
 	}
 
 	/**
-	 * @return mixed
+	 * @return string
 	 */
-	public function getPseudo() {
+	public function getPseudo(): string {
 		return $this->_pseudo;
 	}
 
 	/**
-	 * @return mixed
+	 * @return string
 	 */
-	public function getEmail() {
+	public function getEmail(): string {
 		return $this->_email;
 	}
 
 	/**
-	 * @return mixed
+	 * @return string
 	 */
-	public function getPassword() {
+	public function getPassword(): string {
 		return $this->_password;
 	}
 
 	/**
-	 * @return mixed
+	 * @return string
 	 */
-	public function getCreateAt() {
+	public function getCreateAt(): string {
 		return $this->_create_at;
 	}
 
 	/**
-	 * @return mixed
+	 * @return string
 	 */
-	public function getAvatar() {
+	public function getAvatar(): string {
 		return $this->_avatar;
 	}
 
 	/**
-	 * @param mixed $id
+	 * @param int $id
 	 */
-	public function setId(int $id ) {
+	public function setId(int $id ): void {
```

Comment:
Codacy found an issue: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)